src/olympia/pages/templates/pages/dev_faq.html

```diff
@@ -70,6 +70,13 @@ <h2 id="developing">{{ _('Developing an Add-on') }}</h2>
         of the browser UI.
       {% endtrans %}
     </p>
+    <p>
+      {% trans h_url="https://developer.mozilla.org/Add-ons/WebExtensions",
+      i_url="https://developer.mozilla.org/Add-ons/WebExtensions/Porting_a_legacy_Firefox_add-on"%}
```

Comment:
Indent the second line to match the first (like the 'Are there 3rd party developers that I can hire to build my add-on' section below).  And leave a space before the closing `%}`